rero_ils/modules/documents/api.py

```diff
@@ -174,7 +174,7 @@ def index_persons(self, bulk=False):
             auth_pid = author.get('pid')
             if auth_pid:
                 from ..persons.api import Person
-                person = Person.get_record_by_pid(auth_pid)
+                person = Person.get_record_by_mef_pid(auth_pid)
```

Comment:
Is it related to this commit?

Reply:
It is. I got problems related to person in setup. That's the solution I found.

Reply:
We need to remove all these config in a separate PR and remove also useless js code.